doc/motion_planning_api/src/motion_planning_api_tutorial.cpp

```diff
@@ -306,7 +315,7 @@ int main(int argc, char** argv)
 
   /* But, let's impose a path constraint on the motion.
      Here, we are asking for the end-effector to stay level*/
-  geometry_msgs::QuaternionStamped quaternion;
+  geometry_msgs::msg::QuaternionStamped quaternion;
   quaternion.header.frame_id = "panda_link0";
   quaternion.quaternion.w = 1.0;
```

Comment:
`w` defaults to 1.0 in ROS2 so you can delete this line

Reply:
Fixed.